Tools/python/CMSSWTools.py

```diff
@@ -303,7 +303,7 @@ def is_edm_file(fn):
     return os.system('edmFileUtil %s >/dev/null 2>&1' % fn) == 0
 
 def json_path(bn):
-    return os.path.join(os.environ['CMSSW_BASE'], 'src/JMTucker/MFVNeutralino/test/jsons', bn)
+       return os.path.join(os.environ['CMSSW_BASE'], 'src/JMTucker/MFVNeutralino/test/jsons', bn)
```

Comment:
This change looks like it adds extra indentation that we don't need, let's revert it.

Reply:
Actually, we have more important things to do. It's okay as-is.